MIP102/MIP102c2-Subproposals/MIP102c2-SP18.md

```diff
@@ -34,3 +34,18 @@ This subproposal will amend MIP104 to provide clarifying edits to [recently upda
 ### Amendment Pull Request
 
 https://github.com/makerdao/mips/pull/975
+
```

Comment:
please update the forum link in the preamble to https://forum.makerdao.com/t/mip102c-sp18-mip-amendment-subproposals/22027

Reply:
(That's the same URL)

Reply:
Updated. (The previous URL pointed to the same place [it's resolved by Discourse], but I agree that this one is nicer :) )